### DISCUSSION TITLE
How to use nextjs with react-infinite-scroller

### DISCUSSION THREAD

**Discussion by derekang** [vercel/next.js#12961] [Help] [answered]
I am trying to implement [react-infinite-scroller](https://github.com/CassetteRocks/react-infinite-scroller) into my nextjs app.

Does anyone know of any good tutorials or examples to share? I got stucked at state management.

**Comment by jamesmosier** [comment]
Hi @derekang! Do you have some code you can share?

**Reply by derekang** [reply]
Thanks! I've managed to implement infinite loading feature using useSWRPages instead of react-infinite-scroller. :)

**Comment by derekang** [comment, marked as answer]
Implemented infinite loading feature using swr pagination. https://github.com/zeit/swr/blob/master/examples/pagination/pages/index.js
